Libraries/LibLine/Editor.h

```diff
@@ -293,6 +289,11 @@ class Editor {
     void recalculate_origin();
     void reposition_cursor();
 
+    struct CodepointRange {
+        size_t start { 0 }, end { 0 };
```

Comment:
```suggestion
        size_t start { 0 };
        size_t end { 0 };
```

Would be more Serenity C++ style